src/com/example/powerup/Score.java

```diff
@@ -0,0 +1,8 @@
+package com.example.powerup;
+
+public class Score {
+	public Integer Healing;
+	public Integer Strength;
+	public Integer Telepathy;
+	public Integer Invisibilty;
```

Comment:
You should always be using private attributes and defining getters and setters for them. Use lowercase for attributes (healing, strength, telepathy, invisibility)